chromadb/server/fastapi/__init__.py

```diff
@@ -876,6 +877,11 @@ def process_get(request: Request, raw_body: bytes) -> GetResult:
             ),
         )
 
+        if get_result["embeddings"] is not None:
+            get_result["embeddings"]= [embedding.tolist() if isinstance(embedding, np.ndarray) else embedding for embedding in get_result["embeddings"]]
```

Comment:
When would the get_result not be an np array?

Reply:
In this instance it shouldn't be but this doesn't pass the type checker if we don't check, since `QueryResult` can take a `List[PyEmbeddings]`. We can also cast if that makes more sense to you but thought it reads clearer this way.

Reply:
When we know what type to expect, I prefer that we `cast` to make it clear what the contract is supposed to be.